config/ospd-openvas.service

```
@@ -1,5 +1,5 @@
 [Unit]
-Description=OSPD OpenVAS
+Description=OSPD OpenVAS Wrapper (ospd-openvas)
```

Maybe
```suggestion
Description=OSPd based OpenVAS scanner (ospd-openvas)
```
Personally I am not a fan of the term wrapper.